client/javascript/scrabble.js

```diff
@@ -424,32 +652,95 @@ function Board() {
 
 	    var square = new Square('Normal', this);
 	    
-	    var middle = Math.floor(this.Dimension / 2);
+	    //var middle = Math.floor(this.Dimension / 2);
 	    var halfMiddle = Math.ceil(middle / 2);
-	    
+	    if (this.Dimension == 15) {
+
 	    if ((x == 0 || x == this.Dimension - 1 || x == middle)
```

Comment:
It seems to me that a table-based implementation of the special field locations would now be easier to comprehend, in particular for the SuperSize board.  If you don't feel like it, then please extract the "Regular" and "SuperSize" cases into two separate functions and make sure that the indentation is correct.